<launch>

  <!-- args -->
  <arg name="bag_path"
    default="/home/andre/Documents/rosbags/compact.bag"/>

  <!-- clock -->
  <param name ="/use_sim_time" value="true"/>

  <!-- nodes -->
  <!-- <node name="republish" type="republish" pkg="image_transport" output="screen" -->
  <!--   args="compressed in:=/image_raw raw out:=/image_raw" /> -->
  <node pkg="rosbag" type="play" name="darknet_bag" args="--clock $(arg bag_path)"/>  
  <node pkg="mapper" type="mapper_node" name="mapper" output="screen">
    <param name="img_width"  value="1292"/>
    <param name="img_height" value="964"/>
    <param name="match_box" value="25"/>
    <param name="filter_window" value="5"/>
    <param name="mapper_inc" value="20"/>
  </node>

</launch>
